.travis.yml

```diff
@@ -17,9 +17,9 @@ before_install:
 - export PATH=$MINICONDA_PATH/bin:$PATH
 - conda update --yes conda
 install:
-- conda create --yes -n testenv python=3.6
+- conda env create python=3.6 -f environment.yml --quiet
 - source activate testenv
-- pip install --upgrade sphinx sphinx-sitemap nbsphinx sphinx_rtd_theme ipython
+- jupyter-nbextension enable nglview --py --sys-prefix
 script:
 - make -C ${SPHINX_DIR} html && touch ${HTML_DIR}/.nojekyll
 deploy:
```

Comment:
Kill the DS Store file below

Reply:
Add a top-level `.gitignore` ! You can copy the MDA [.gitignore](https://github.com/MDAnalysis/mdanalysis/blob/develop/.gitignore)